doc/user_manual/generated/internalRom.tex

```diff
@@ -11122,7 +11122,7 @@ \subsubsection{DMDBase}
           performed.
   \default{0}
 
-        \item \xmlNode{normalize\_y}: \xmlDesc{[True, Yes, 1, False, No, 0, t, y, 1, f, n, 0]},
+        \item \xmlNode{normalize\_y}: \xmlDesc{[True, Yes, 1, False, No, 0, t, y, 1, f, n, 0]}, 
```

Comment:
I do not see any obvious changes except added extra space at  the end of the line. Please try to clean the end line spaces, and see if there is any changes that you want to change. It seems to me you do not need to change this file "internalRom.tex" for this PR.

Reply:
I removed all trailing whitespace in internalRom.tex and optimizer.tex.

Reply:
If you would like the whitespace removal as a separate pull request, let me know. Thanks.

Reply:
Here is the branch with just a whitespace cleanup: https://github.com/joshua-cogliati-inl/raven/tree/whitespace_cleanup

Reply:
Fixed with the merge of #2453